packages/tools/cookbook/package.json

```diff
@@ -12,10 +12,10 @@
     "build:local": "pnpm run pactjs:coin && heft build --clean",
     "format": "pnpm run /^format:.*/",
     "format:md": "remark README.md -o --use @kadena-dev/markdown",
-    "format:src": "prettier . --cache --write",
+    "format:lint": "pnpm run lint:src --fix","format:src": "prettier . --cache --write",
```

Comment:
Commands should be on separate lines?

Reply:
Definitely! Guess those were skipped because of other failing formatting (see desc.). Fixed.